zds/settings.py

```diff
@@ -366,7 +365,6 @@
 # Fake mails (in console)
 EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
-from django.contrib.messages import constants as message_constants
```

Comment:
`Flake8 Error E402 — module level import not at top of fileat line 369 col 1`

Et déjà présent à la ligne 7 de la nouvelle version ou 8 de l'ancienne version.